connector_magento_multi_source_inventory/components/binder.py

```diff
@@ -0,0 +1,33 @@
+# © 2013 Guewen Baconnier,Camptocamp SA,Akretion
+# © 2016 Sodexis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.addons.component.core import Component
+
+
+class MagentoModelBinder(Component):
+    """ Bind records and give odoo/magento ids correspondence
+
+    Binding models are models called ``magento.{normal_model}``,
+    like ``magento.res.partner`` or ``magento.product.product``.
+    They are ``_inherits`` of the normal models and contains
+    the Magento ID, the ID of the Magento Backend and the additional
+    fields belonging to the Magento instance.
+    """
+    _name = 'magento.binder'
```

Comment:
I'm pretty sure you are not supposed to list all known `apply_on`s below. Would it work if you leave out _name here, inherit from 'magento.binder' instead and just set `_apply_on = 'magento.source`'?